development/webpack/webpack.config.ts

```diff
@@ -230,6 +230,11 @@ const config = {
   resolve: {
     // Disable symlinks for performance; saves about .5 seconds off full build
     symlinks: false,
+    // Explicitly alias JSX runtimes to avoid ESM fully-specified resolution issues
+    alias: {
+      'react/jsx-runtime': require.resolve('react/jsx-runtime.js'),
+      'react/jsx-dev-runtime': require.resolve('react/jsx-dev-runtime.js'),
+    },
```

Comment:
- Radix’s ESM build imports the subpath 'react/jsx-runtime'. In strict ESM, webpack treats the request as “fully specified” and may fail to resolve the bare subpath (especially with nested deps/exports maps). It then asks for an explicit file with extension.
- Forces those subpath imports to resolve directly to the actual files in React (…/react/jsx-runtime.js and …/react/jsx-dev-runtime.js), bypassing exports-map ambiguity and nested node_modules resolution.
- Webpack no longer errors on “Can’t resolve 'react/jsx-runtime' / fully specified ESM.”